data/docs/instrumentation/opentelemetry-python/overview.mdx

```diff
@@ -0,0 +1,588 @@
+```
+
+Then set `OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf`.
+
+### Database calls not showing in traces
+
+Auto-instrumentation detects and instruments common database libraries. Ensure you've run `opentelemetry-bootstrap --action=install` after installing your database drivers.
+
+**PostgreSQL note**: `psycopg2` is preferred over `psycopg2-binary` for auto-instrumentation. If you must use `psycopg2-binary`, you may need to use manual instrumentation with `Psycopg2Instrumentor().instrument(skip_dep_check=True)`.
+
+Common database instrumentations installed by bootstrap:
+- PostgreSQL: `opentelemetry-instrumentation-psycopg2`
+- MySQL: `opentelemetry-instrumentation-pymysql` or `opentelemetry-instrumentation-mysql`
+- MongoDB: `opentelemetry-instrumentation-pymongo`
+- Redis: `opentelemetry-instrumentation-redis`
+- SQLAlchemy: `opentelemetry-instrumentation-sqlalchemy`
+
+Check <a href="https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation" target="_blank" rel="noopener noreferrer nofollow">supported versions</a> for compatibility with your library versions.
+
+</details>
+
+<details>
+<ToggleHeading>
+
+## Setup OpenTelemetry Collector (Optional)
+
+</ToggleHeading>
+
+### What is the OpenTelemetry Collector?
+
+Think of the OTel Collector as a middleman between your app and SigNoz. Instead of your application sending data directly to SigNoz, it sends everything to the Collector first, which then forwards it along.
+
+### Why use it?
+
+- **Cleaning up data** — Filter out noisy traces you don't care about, or remove sensitive info before it leaves your servers.
+- **Keeping your app lightweight** — Let the Collector handle batching, retries, and compression instead of your application code.
+- **Adding context automatically** — The Collector can tag your data with useful info like which Kubernetes pod or cloud region it came from.
+- **Future flexibility** — Want to send data to multiple backends later? The Collector makes that easy without changing your app.
+
+See [Switch from direct export to Collector](https://signoz.io/docs/opentelemetry-collection-agents/opentelemetry-collector/switch-to-collector/) for step-by-step instructions to convert your setup.
+
+For more details, see [Why use the OpenTelemetry Collector?](https://signoz.io/docs/opentelemetry-collection-agents/opentelemetry-collector/why-to-use-collector/) and the [Collector configuration guide](https://signoz.io/docs/opentelemetry-collection-agents/opentelemetry-collector/configuration/).
+
+</details>
+
+## Next steps
+
+- [Add manual instrumentation](https://signoz.io/docs/instrumentation/manual-instrumentation/python/manual-instrumentation) for custom spans and attributes
+- Send logs from Python ([auto-instrumentation](https://signoz.io/docs/userguide/python-logs-auto-instrumentation/) | [OTel SDK](https://signoz.io/docs/userguide/collecting_application_logs_otel_sdk_python/)) and correlate with traces
```

Comment:
Remove this [OTel SDK](https://signoz.io/docs/userguide/collecting_application_logs_otel_sdk_python/))